src/get_mounts/mod.rs

```diff
@@ -1,9 +1,9 @@
-cfg_if! {
-        if #[cfg(target_os = "linux")] {
-                mod getmntent;
-                pub use self::getmntent::get_mount_points;
-        } else {
-                mod getmntinfo;
-                pub use self::getmntinfo::get_mount_points;
-        }
-}
+#[cfg(target_os = "linux")]
+mod linux;
```

Comment:
This is a somewhat philosophical question: I used the API names for these modules because `getmntent` is only used on Linux but `getmntinfo` was used by Darwin & Solaris in addition to various BSDs.

Reply:
I tend to consider Darwin basically a BSD, but how about something like `linux` and `other_unix`?

Reply:
Thinking about this more, I'm +1 on just calling it BSD since that's the origin of that particular API even if the kernel implementation on XNU has likely drifted pretty far by now.